js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -808,3 +808,21 @@ invalidateRotatedSecret=Das rotierte Secret invalidieren?
 noDirectUsers=Keine direkten Benutzer
 permissionsDisable=Berechtigung abschalten?
 eventTypes.PERMISSION_TOKEN.name=Berechtigungstoken
+eventTypes.UPDATE_TOTP.name=Update totp
+subjectNameId=Betreff NameID
+serviceProviderEntityId=Service Provider Entity ID
+artifactBindingUrl=Artifact Binding URL
+eventTypes.LOGIN.name=Anmelden
+clearFileExplain=Sind Sie sicher, dass Sie diese Datei leeren möchten?
+eventTypes.LOGOUT_ERROR.name=Logout Fehler
+allowRemoteResourceManagement=Remote-Ressourcenverwaltung
+expirationValueNotValid=Wert sollte größer oder gleich 1 sein
+eventTypes.UPDATE_CONSENT.name=Aktualisierung der Einwilligung
+forceArtifactBinding=Artifact Binding erzwingen
+eventTypes.REFRESH_TOKEN_ERROR.description=Refresh Token Fehler
+eventTypes.IMPERSONATE.name=Impersonieren
```

Comment:
Not sure if there is a good German word for this. What about `Sich ausgeben als`? The context where this is used: An admin can impersonate a user and then act as this user for example to verify a problem the user is reporting.